src/live.asp.net/Controllers/HomeController.cs

```diff
@@ -37,6 +37,14 @@ public HomeController(IShowsService showsService, ILiveShowDetailsService liveSh
             });
         }
 
+        [HttpGet("/feed")]
+        [Produces("application/rss+xml")]
+        public async Task<string> GetRssFeed()
+        {
+            var feedService = new FeedService(_showsService, false);
```

Comment:
I was trying to use the same thing as in the `Index` action. I couldn't understand where `disableCache` was coming from.